lib/mage/Mage.js

```diff
@@ -817,23 +819,21 @@ Mage.prototype.isDevelopmentMode = function (feature) {
 };
 
 
-Mage.prototype.getClientConfig = function (appName, baseUrl) {
+Mage.prototype.getConfig = function (appName) {
```

Comment:
We most likely need to still have the baseUrl to be injectable for dynamic environments (example: CI deployment)